docker/images/proxysql/suse-compliant/entrypoint/entrypoint.bash

```
@@ -84,10 +84,41 @@ cp -a systemd proxysql-${CURVER}/etc/
+    fi
+fi
+
+# Belt-and-braces: only set with_plugins=1 when at least one .so was
+# actually staged.  rpmbuild aborts on "File not found by glob" if the
+# %files block lists /usr/lib/proxysql/*.so and the directory is empty.
+RPMBUILD_WITH_PLUGINS=0
+if compgen -G "proxysql-${CURVER}/usr/lib/proxysql/*.so" >/dev/null 2>&1; then
+    RPMBUILD_WITH_PLUGINS=1
+fi
+
 tar czvf "proxysql-${CURVER}.tar.gz" proxysql-${CURVER}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)